src/utils/cspr-transfer.ts

```diff
@@ -105,15 +109,16 @@ export const makeCsprTransferTransaction = ({
   memo,
   ttl = DEFAULT_DEPLOY_TTL,
   timestamp,
-  casperNetworkApiVersion
+  casperNetworkApiVersion,
+  gasPrice = 1
 }: IMakeCsprTransferTransactionParams): Transaction => {
   if (casperNetworkApiVersion.startsWith('2')) {
     let txBuilder = new NativeTransferBuilder()
       .from(PublicKey.fromHex(senderPublicKeyHex))
       .target(PublicKey.fromHex(recipientPublicKeyHex))
       .amount(transferAmount)
       .chainName(chainName)
-      .payment(100000000)
+      .payment(100000000, gasPrice)
```

Comment:
consider to make this an input argument with default value (as with auction calls)